<!-- This is an example launch file for how a ROS application could use a node -->
<launch>

	<arg name="instance_id" default="default"/>
	<param name="instance_id" value="$(arg instance_id)"/>
    <rosparam file="$(find harmoni_openface)/config/configuration.yaml" subst_value="True"/>
    <node pkg="harmoni_openface" type="openface_service.py" name="harmoni_openface_$(arg instance_id)" output="screen"/>

</launch>
